pkg/service/rulesvc/create.go

```diff
@@ -143,11 +181,22 @@ func (s *Service) CreateAccessRule(ctx context.Context, userID string, in types.
 	log := logger.Get(ctx).With("user.id", userID, "access_rule.id", id)
 	now := s.Clock.Now()
 
-	target, err := s.ProcessTarget(ctx, in.Target)
-	if err != nil {
+	q := storage.GetTargetGroup{ID: in.Target.ProviderId}
+	_, err := s.DB.Query(ctx, &q)
+	if err != nil && err != ddb.ErrNoItems {
 		return nil, err
 	}
 
+	var target rule.Target
+	if q.Result.ID != "" {
```

Comment:
Can you replace this if statement by just passing a boolean into the function
Seems like the target group query should return a pointer but its not,